test/e2e_node/eviction_test.go

```diff
@@ -500,15 +501,38 @@ var _ = SIGDescribe("PriorityPidEvictionOrdering [Slow] [Serial] [Disruptive][No
 		specs[2].pod.Spec.PriorityClassName = highPriorityClassName
 		runEvictionTest(f, pressureTimeout, expectedNodeCondition, expectedStarvedResource, logPidMetrics, specs)
 	})
+
+	ginkgo.Context(fmt.Sprintf(testContextFmt, expectedNodeCondition)+"; PodDisruptionConditions enabled [Feature:PodDisruptionConditions]", func() {
```

Comment:
I think you likely need `NodeFeature:` here not, `Feature:` since the node e2e tests use a focus of `NodeFeature:`

Reply:
yeah, I'm confused about the naming. Some other features also use just "Feature:" prefix, for example: [Feature:CPUManager]"

Reply:
This is because CPU manager has a special job which targets `Feature:CPUManager` - https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-node/containerd.yaml#L1185-L1210

For the rest of the e2e jobs, we use NodeFeature: https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/kubernetes/test-infra%24+file:%5Econfig/jobs/kubernetes/sig-node/containerd%5C.yaml+NodeFeature&patternType=standard

Reply:
actually, I also prepared a dedicated PR to test-infra: https://github.com/kubernetes/test-infra/pull/27897 targeting `Feature:`, but this can also be `NodeFeature:`. Still the test will remain not to run as it inherits the [NodeFeature:Eviction] tag from the SIGDescribe